File: clientepython/model/Libro.py
from datetime import datetime

class Libro:
    def __init__(self, titulo: str, autor: str, cantidad_paginas: int, 
                 fecha_creacion: datetime, precio: float, tapa_dura: bool):
        self.titulo = titulo
        self.autor = autor
        self.cantidad_paginas = cantidad_paginas
        self.fecha_creacion = fecha_creacion
        self.precio = precio
        self.tapa_dura = tapa_dura

File: ClientePublicacion/Cliente/Cliente/Model/Libro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cliente.Model
{
    public class Libro
    {
        public string Titulo { get; set; }
        public string Autor { get; set; }
        public int CantidadPaginas { get; set; }
        public DateTime FechaCreacion { get; set; }
        public double Precio { get; set; }
        public bool TapaDura { get; set; }

        public Libro(string titulo, String autor, int cantidadPaginas, DateTime fechaCreacion, double precio, bool tapaDura)
        {
            Titulo = titulo;
            Autor = autor;
            CantidadPaginas = cantidadPaginas;
            FechaCreacion = fechaCreacion;
            Precio = precio;
            TapaDura = tapaDura;
        }
    }
}
